Flashcards.harris-andy/Classes/UseDB.cs

```diff
@@ -0,0 +1,133 @@
+using Microsoft.Data.SqlClient;
+using Dapper;
+using Flashcards.harris_andy.Classes;
+
+namespace Flashcards.harris_andy;
+
+public class UseDB
+{
+    public void InitializeDatabase()
+    {
+        string connectionStringMaster = "Server=localhost,1433;Database=master;User Id=SA;Password=SuperStrongSexyPassword123;TrustServerCertificate=True;";
```

Comment:
I did it this way because there are two different connection strings. This one, connectionStringMaster is to create the database (using database "master"). Then once it's created I used the app.config class to more concisely refer to the flash cards db connection string.